lib/screen/note_screen.dart

```diff
@@ -336,6 +344,37 @@ class _NoteScreen extends State<NoteScreen> with CommandHandler {
         ? Tuple2(notes.sublist(0, indexUnpinned), notes.sublist(indexUnpinned))
         : Tuple2(notes, []);
   }
+
+  void _deleteBook() async {
+    final uid = FirebaseAuth.instance.currentUser.uid;
+    final yes = await showDialog<bool>(
+      context: context,
+      builder: (context) => AlertDialog(
+        content: const Text('Are you sure to delete this book ?'),
+        actions: <Widget>[
+          TextButton(
+            child: const Text('No'),
+            onPressed: () => Navigator.pop(context, false),
+          ),
+          TextButton(
+            child: const Text('Yes'),
+            onPressed: () => Navigator.pop(context, true),
```

Comment:
Or you may don't need to keep the user response in a variable but instead, move the if block below (`if (yes){...}`) in this "yes" block here.